src/tilesource.js

```diff
@@ -420,6 +423,41 @@ $.TileSource.prototype = {
         return new $.Rect( px * scale, py * scale, sx * scale, sy * scale );
     },
 
+    /**
+     * @function
+     * @param {Number} level
+     * @param {Number} x
+     * @param {Number} y
+     * @param {Boolean} [isSource=false] Whether to return the source bounds of the tile.
+     * @returns {OpenSeadragon.Rect} Either where this tile fits (in normalized coordinates unless imageCoordinates == true) or the
```

Comment:
What are normalized coordinates? How do they relate to our other coordinate systems?

Reply:
Getting rid of this entire function, that was a leftover from supporting my earlier three.js implementation.